GreenGarbageApp/app/src/main/res/layout/fragment_game.xml

```diff
@@ -53,6 +53,12 @@
             android:background="@color/white"
             android:orientation="horizontal">
 
```

Comment:
This space is currently full, after Umair added the trash count here. We need to move this to somewhere else.

Reply:
I'll add a new linear layout.